tests/common/common_python_vec.py

```diff
@@ -192,6 +219,12 @@ class ReverseData:
 }
 """)
 
+def clear_type_str(type_str):
+    """
+    Clear type name from from namespaces ans whitspaces
```

Comment:
Spell checker...

Reply:
Perhaps the function name could be more understandable.

Reply:
Fixed spelling. Renamed to  `remove_namespaces_whitspaces`